ci/ray_ci/test_tester.py

```diff
@@ -84,6 +85,11 @@ def test_get_test_targets() -> None:
             ]
 
 
+def test_add_default_except_tags() -> None:
+    assert _add_default_except_tags("tag1,tag2") == "tag1,tag2,manual"
+    assert _add_default_except_tags("") == "manual"
```

Comment:
what will happen for `_add_default_except_tags("manual")`? should we clean it and make it a set?

Reply:
lol good catch